src/client/components/menu/Menu.component.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Menu.styles.css';
+import faUser from '../../assets/images/user-login.png';
+import faHeart from '../../assets/images/favorite-icon.png';
+import faShoppingCart from '../../assets/images/shopping-cart.png';
+
+export const Menu = () => {
+  return (
+    <menu>
+      <header>
+        <div className="logo_container">
+          <div className="user-icon">
+            <div className="dropdown">
+              <img src={faUser} alt="login" />
+
+              <div id="login" className="dropdown-content">
+                <a href="login">LOGIN / SIGNUP</a>
```

Comment:
For navigation in react, use routing. In your case, you will need to use `Link` . You can read about it [here](https://v5.reactrouter.com/native/api/Link)

Reply:
@valmdz Thanks for letting me know . I changed it please review.

Reply:
@valmdz I'm resolving any of your comments that I see fixed.